doc/api/errors.md

```diff
@@ -471,7 +471,24 @@ of error codes and their meanings is available by running `man 2 intro` or
 In Node.js, system errors are represented as augmented `Error` objects with
 added properties.
 
-### Class: System Error
+### Class: SystemError
+
+### error.info
+
+`SystemError` instances may have an additional `info` property whose
```

Comment:
Come to think of it, I think a more popular pattern in the userland is using the `err.data` property, I remember seeing some logging services collect that automatically in addition to `err.code`...I remember there is another PR for crypto that use `info`. Is there a reason that we pick `info` instead of `data`?

Reply:
A quick search in our own code base shows that eslint is using `error.data`: https://github.com/nodejs/node/blob/master/tools/eslint/lib/config/config-validator.js#L92 and https://github.com/nodejs/node/blob/master/tools/eslint/lib/config/config-validator.js#L179

Reply:
I've seen both: https://www.npmjs.com/package/verror uses `*.info`

Reply:
On a side note: would be easier to read if we have a syntax for documenting optional properties..

Reply:
I agree, definitely something worth working on